sebs/types.py

```
@@ -21,8 +22,39 @@ class Storage(str, Enum):
+    X86 = "x64"
+    ARM = "arm64"
+
+    def serialize(self) -> str:
+        return self.value
+
+    @staticmethod
+    def deserialize(val: str) -> Architecture:
+        for member in Architecture:
+            if member.value == val:
+                return member
+        raise Exception(f"Unknown architecture type {member}")
```

Ruff (0.14.7)

Create your own exception

(TRY002)